internal/translation/encryptionatrest/conversion.go

```diff
@@ -0,0 +1,175 @@
+package encryptionatrest
+
+import (
+	"reflect"
+
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+
+	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
+)
+
+type EncryptionAtRest struct {
+	AWS   AwsKms
+	Azure AzureKeyVault
+	GCP   GoogleCloudKms
+}
+
+type AwsKms struct {
+	akov2.AwsKms
+	AccessKeyID                  string
+	SecretAccessKey              string
```

Comment:
These 2 are never sent nor received, are they needed?

Reply:
These are fields that Atlas seems to expect, but we don't seem to currently include. I don't want to add these fields (especially if they are mandatory) without further discussion with the team, but they exist in the API/SDK and are 'sensitive' fields.

Reply:
I see, I think this merits a comment explaining the situation and maybe even a link to theAPI endpoint docs. Because looking at the rest of the code it is not obvious why these are here.

Reply:
Setups by access/secret keys are discouraged and I believe API doesn't accept it anymore, it's was kept to avoid breaking change, if I'm not wrong.